### PR TITLE
bsdlib: version 0.7.5

### PR DESCRIPTION
**Pull request by evenl**
Library version: 537db4caa39f7820b4df290d32953e9e128204de

 * Updated bsd_shutdown() to perform a proper shutdown of the modem and the
   library.
 * Updated bsd_init() to properly support multiple initializations of
   the modem and library.
 * New socket options added:
   SILENCE_ALL, SILENCE_IP_ECHO_REPLY, SILENCE_IPV6_ECHO_REPLY and REUSEADDR
 * Fix to fidoless trace disable 

Manifest update: https://github.com/nrfconnect/sdk-nrf/pull/2542
Zephyr PR: https://github.com/nrfconnect/sdk-zephyr/pull/325

Signed-off-by: Even Falch-Larsen <even.falch-larsen@nordicsemi.no>
Signed-off-by: Petri Honkala <petri.honkala@nordicsemi.no>

### CODE REVIEW

**Comment by rlubos** [comment]
@evenl Could you please add 0.7.4 changelog to this PR, so we can close the other one?

**Comment by evenl** [comment]
Merged PR#217 into this, and fixed some documentation update issue due to not beeing backported to sampo.

**Comment by b-gent** [comment]
Hi @umapraseeda  is now assigned as a technical writer for the IoT team, so I'm reassigning to her. Please assign her also in future releases.

File: bsdlib/doc/CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,21 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.7.5
+************
+
+* Updated `bsd_shutdown()` to perform a proper shutdown of the modem and the
+  library.
```
Comment (umapraseeda) [inline bsdlib/doc/CHANGELOG.rst:12]:
```suggestion
* Updated `bsd_shutdown()` to perform a proper shutdown of the modem and the library.
```

File: bsdlib/doc/CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,21 @@ Changelog
...
+* Updated `bsd_init()` to properly support multiple initializations of the
+  modem and library.
```
Comment (umapraseeda) [inline bsdlib/doc/CHANGELOG.rst:14]:
```suggestion
* Updated `bsd_init()` to properly support multiple initializations of the modem and the library.
```

**Pull request merged**